### PR TITLE
chore(short-survey): MDN Web Docs content discovery

### LINKED ISSUE #8314: chore(short-survey): MDN Web Docs content discovery
The MDN Web Docs authors team would like to understand the needs of our readers. We want to run a single-question short survey for the week of 27 February 2023.

### PR DESCRIPTION
**Pull request by schalkneethling**
Adds a short survey to determine the content needs of our readers. The survey will be
shown to 50% of our audience for the week of 27 February 2023.

### Initial state

Shows the survey in its collapsed state. Shows title and call to action.

### Expanded state

Shows the survey in its expanded state. Shows title, call to action, and a text input field.

fix #8314

### CODE REVIEW

File: client/src/ui/molecules/document-survey/surveys.ts [hunk 1]
```diff
@@ -35,15 +37,14 @@ enum SurveyKey {
...
+    bucket: SurveyBucket.CONTENT_DISCOVERY_2023,
+    show: (doc: Doc) => /en-US\/docs\/Web(\/|$)/i.test(doc.mdn_url),
+    src: "https://survey.alchemer.com/s3/7241049/MDN-Web-Docs-Content-Discovery",
     teaser:
-      "Ensuring the web is open and accessible to all is central to Mozilla’s mission. The Firefox product team is interested in learning which features are most important to support across all major browsers.",
-    question: "Let us know what you think.",
-    ...survey_duration(SurveyBucket.FIREFOX_WEB_COMPAT_2023),
-    ...survey_rates(SurveyKey.FIREFOX_WEB_COMPAT_2023),
+      "The MDN Web Docs writers team wants to understand the needs of our readers better.",
+    question: "Help us shape the future of MDN Web Docs.",
```
Comment (bsmth) [inline client/src/ui/molecules/document-survey/surveys.ts:46]:
```suggestion
    question: "Help us understand the content you're interested in seeing.",
```

File: client/src/ui/molecules/document-survey/surveys.ts [hunk 2]
```diff
@@ -35,15 +37,14 @@ enum SurveyKey {
...
+    key: SurveyKey.CONTENT_DISCOVERY_2023,
+    bucket: SurveyBucket.CONTENT_DISCOVERY_2023,
+    show: (doc: Doc) => /en-US\/docs\/Web(\/|$)/i.test(doc.mdn_url),
+    src: "https://survey.alchemer.com/s3/7241049/MDN-Web-Docs-Content-Discovery",
     teaser:
-      "Ensuring the web is open and accessible to all is central to Mozilla’s mission. The Firefox product team is interested in learning which features are most important to support across all major browsers.",
-    question: "Let us know what you think.",
-    ...survey_duration(SurveyBucket.FIREFOX_WEB_COMPAT_2023),
-    ...survey_rates(SurveyKey.FIREFOX_WEB_COMPAT_2023),
+      "The MDN Web Docs writers team wants to understand the needs of our readers better.",
```
[the remark this answers is not in the document]
Comment (schalkneethling) [reply client/src/ui/molecules/document-survey/surveys.ts:45]: @Rumyra, this looks good to me. Thoughts?

File: .github/workflows/prod-build.yml [hunk 3]
```diff
@@ -170,10 +170,10 @@ jobs:
...
 
           # Surveys.
-          REACT_APP_SURVEY_START_FIREFOX_WEB_COMPAT_2023: 1675252800000 # (new Date("2023-02-01 12:00:00Z")).getTime()
-          REACT_APP_SURVEY_END_FIREFOX_WEB_COMPAT_2023: 1675944000000 # (new Date("2023-02-09 12:00:00Z")).getTime()
-          REACT_APP_SURVEY_RATE_FROM_FIREFOX_WEB_COMPAT_2023: 0.0
-          REACT_APP_SURVEY_RATE_TILL_FIREFOX_WEB_COMPAT_2023: 0.10 # 10%
+          REACT_APP_SURVEY_START_CONTENT_DISCOVERY_2023: 1677499200000 # (new Date("2023-02-27 12:00:00Z")).getTime()
+          REACT_APP_SURVEY_END_CONTENT_DISCOVERY_2023: 1677672000000 # (new Date("2023-03-01 12:00:00Z")).getTime()
+          REACT_APP_SURVEY_RATE_FROM_CONTENT_DISCOVERY_2023: 0.0
+          REACT_APP_SURVEY_RATE_TILL_CONTENT_DISCOVERY_2023: 0.50 # 50%
```
Comment (schalkneethling) [inline .github/workflows/prod-build.yml:176]:
```suggestion
          REACT_APP_SURVEY_RATE_TILL_CONTENT_DISCOVERY_2023: 0.5 # 5%
```

File: .github/workflows/stage-build.yml [hunk 4]
```diff
@@ -167,10 +167,10 @@ jobs:
...
 
           # Surveys.
-          REACT_APP_SURVEY_START_FIREFOX_WEB_COMPAT_2023: 0 # stage
-          REACT_APP_SURVEY_END_FIREFOX_WEB_COMPAT_2023: 1675944000000 # (new Date("2023-02-09 12:00:00Z")).getTime()
-          REACT_APP_SURVEY_RATE_FROM_FIREFOX_WEB_COMPAT_2023: 0.0
-          REACT_APP_SURVEY_RATE_TILL_FIREFOX_WEB_COMPAT_2023: 0.10 # 10%
+          REACT_APP_SURVEY_START_CONTENT_DISCOVERY_2023: 0 # stage
+          REACT_APP_SURVEY_END_CONTENT_DISCOVERY_2023: 1677672000000 # (new Date("2023-03-01 12:00:00Z")).getTime()
+          REACT_APP_SURVEY_RATE_FROM_CONTENT_DISCOVERY_2023: 0.0
+          REACT_APP_SURVEY_RATE_TILL_CONTENT_DISCOVERY_2023: 0.50 # 50%
```
Comment (schalkneethling) [inline .github/workflows/stage-build.yml:173]:
```suggestion
          REACT_APP_SURVEY_RATE_TILL_CONTENT_DISCOVERY_2023: 0.5 # 5%
```

**Review by bsmth — APPROVED** [review]
LGTM

**Comment by Rumyra** [comment]
Yeh LGTM too 👍

File: .github/workflows/prod-build.yml [hunk 5]
```diff
@@ -170,10 +170,10 @@ jobs:
...
 
           # Surveys.
-          REACT_APP_SURVEY_START_FIREFOX_WEB_COMPAT_2023: 1675252800000 # (new Date("2023-02-01 12:00:00Z")).getTime()
-          REACT_APP_SURVEY_END_FIREFOX_WEB_COMPAT_2023: 1675944000000 # (new Date("2023-02-09 12:00:00Z")).getTime()
-          REACT_APP_SURVEY_RATE_FROM_FIREFOX_WEB_COMPAT_2023: 0.0
-          REACT_APP_SURVEY_RATE_TILL_FIREFOX_WEB_COMPAT_2023: 0.10 # 10%
+          REACT_APP_SURVEY_START_CONTENT_DISCOVERY_2023: 1677499200000 # (new Date("2023-02-27 12:00:00Z")).getTime()
+          REACT_APP_SURVEY_END_CONTENT_DISCOVERY_2023: 1677672000000 # (new Date("2023-03-01 12:00:00Z")).getTime()
+          REACT_APP_SURVEY_RATE_FROM_CONTENT_DISCOVERY_2023: 0.0
+          REACT_APP_SURVEY_RATE_TILL_CONTENT_DISCOVERY_2023: 0.5 # 5%
```
Comment (caugner) [inline .github/workflows/prod-build.yml:176]:
```suggestion
          REACT_APP_SURVEY_RATE_TILL_CONTENT_DISCOVERY_2023: 0.05 # 5%
```

File: .github/workflows/stage-build.yml [hunk 6]
```diff
@@ -167,10 +167,10 @@ jobs:
...
 
           # Surveys.
-          REACT_APP_SURVEY_START_FIREFOX_WEB_COMPAT_2023: 0 # stage
-          REACT_APP_SURVEY_END_FIREFOX_WEB_COMPAT_2023: 1675944000000 # (new Date("2023-02-09 12:00:00Z")).getTime()
-          REACT_APP_SURVEY_RATE_FROM_FIREFOX_WEB_COMPAT_2023: 0.0
-          REACT_APP_SURVEY_RATE_TILL_FIREFOX_WEB_COMPAT_2023: 0.10 # 10%
+          REACT_APP_SURVEY_START_CONTENT_DISCOVERY_2023: 0 # stage
+          REACT_APP_SURVEY_END_CONTENT_DISCOVERY_2023: 1677672000000 # (new Date("2023-03-01 12:00:00Z")).getTime()
+          REACT_APP_SURVEY_RATE_FROM_CONTENT_DISCOVERY_2023: 0.0
+          REACT_APP_SURVEY_RATE_TILL_CONTENT_DISCOVERY_2023: 0.5 # 5%
```
Comment (caugner) [inline .github/workflows/stage-build.yml:173]:
```suggestion
          REACT_APP_SURVEY_RATE_TILL_CONTENT_DISCOVERY_2023: 0.05 # 5%
```

**Review by caugner — CHANGES_REQUESTED** [review]
LGTM, just one nit: `0.5` = 50%

**Review by caugner — APPROVED** [review]

**Comment by schalkneethling** [comment]
> LGTM, just one nit: `0.5` = 50%

Doh! Thanks for fixing my typo.

**Pull request merged**